AVsitter2/Plugins/AVvip/README.md

```diff
@@ -0,0 +1,23 @@
+# AVsitter VIP
```

Comment:
Examples and documentation should be submitted to the https://github.com/avsitter/avsitter.github.io repository. Their place is not the plugin folder.

Reply:
Documentation removed from plugin directory in https://github.com/AVsitter/AVsitter/pull/115/commits/ffdd1ce6e06c5190f60414e9f80efa1d2dd58453